### ISSUE TITLE
xcube docker image is not working

### ISSUE THREAD

**Issue by sfalquier** [dcs4cop/xcube#651]
**Describe the bug**
Following the [documentation](https://xcube.readthedocs.io/en/latest/installation.html#docker), after building xcube docker image from repository (v0.10.2), we are not able to run directly xcube commands in container and bound port does not correspond with image's exposed port.

**To Reproduce**
```
$ docker build -t xcube:latest .
$ docker run -d -p 8001:8000 xcube:latest xcube
8635633f3b9b122d2ee3836cf4afe91af2f486933c4403c22a041475d3637495
docker: Error response from daemon: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "xcube": executable file not found in $PATH: unknown.
```
It looks like `source activate xcube` is not taken into account when command is executed at runtime in the container.

**Expected behavior**
Something like this with correct port mapping : 
```
$ docker build -t xcube:latest .
$ docker run -d -p **8080:8080** xcube:latest xcube
8635633f3b9b122d2ee3836cf4afe91af2f486933c4403c22a041475d3637495
Usage: xcube [OPTIONS] COMMAND [ARGS]...

  xcube Toolkit

Options:
  --version              Show the version and exit.
  --traceback            Enable tracing back errors by dumping the Python call
                         stack. Pass as very first option to also trace back
                         error during command-line validation.
  --scheduler SCHEDULER  Enable distributed computing using the Dask scheduler
                         identified by SCHEDULER. SCHEDULER can have the form
                         <address>?<keyword>=<value>,... where <address> is
                         <host> or <host>:<port> and specifies the scheduler's
                         address in your network. For more information on
                         distributed computing using Dask, refer to
                         http://distributed.dask.org/. Pairs of
                         <keyword>=<value> are passed to the Dask client.
                         Refer to http://distributed.dask.org/en/latest/api.ht
                         ml#distributed.Client
  -w, --warnings         Show any warnings produced during operation (warnings
                         are hidden by default)
  --help                 Show this message and exit.

Commands:
  chunk     (Re-)chunk xcube dataset.
  compute   Compute a cube from one or more other cubes.
  dump      Dump contents of an input dataset.
  edit      Edit the metadata of an xcube dataset.
  extract   Extract cube points.
  gen       Generate xcube dataset.
  grid      Find spatial xcube dataset resolutions and adjust bounding...
  level     Generate multi-resolution levels.
  optimize  Optimize xcube dataset for faster access.
  prune     Delete empty chunks.
  resample  Resample data along the time dimension.
  serve     Serve data cubes via web service.
  sh        SentinelHub tools for xcube.
  tile      Create RGBA tiles from CUBE.
  vars2dim  Convert cube variables into new dimension.
  verify    Perform cube verification.
  versions  Get versions of important packages used by xcube.
```

**Comment by forman** [comment]
Hi @sfalquier, thanks for reporting. We'll check this next week.

**Comment by TonioF** [comment]
Solved by #664.
